### PR TITLE
FAQ for avoiding slow Android builds

### PR DESCRIPTION
**Pull request by advayDev1**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -98,6 +98,19 @@ call even if you do not need to customize any other `j2objConfig` option.
...
 
 
+### Why is my Android build so much slower after adding j2objc?
+
+Depending on your Android Studio 'run configuration', you may be building more of your Gradle
+project than is neccessary.  For example if your run configuration includes a general 'Make'
+task before running, it will build all your gradle projects, including the j2objc parts.
+Instead make sure your run configuration only performs the `:android:assembleDebug` task:
+
+<img width="518" alt="Run Configuration" src="https://cloud.githubusercontent.com/assets/11729521/8971728/0585e344-3608-11e5-9c44-e06b6ad8f48a.png">
```
Comment (brunobowden) [inline FAQ.md]: Can you highlight the "Run Gradle task...." to make the screenshot even more obvious.

**Comment by brunobowden** [comment]
LGTM

Thanks for adding this - just see the comment

**Comment by advayDev1** [comment]
done and merging

**Comment by brunobowden** [comment]
It's fine as it is but I was thinking more of something like this:

[image: 95a511a4-3881-11e5-8525-9e4256990614.png]

**Comment by advayDev1** [comment]
@brunobowden ?  i don't see anything in your response.  i did the highlight as you asked.

**Pull request merged**